docs/ref/contrib/contenttypes.txt

```diff
@@ -391,6 +391,36 @@ be used to retrieve their associated ``TaggedItems``::
     >>> b.tags.all()
     <QuerySet [<TaggedItem: django>, <TaggedItem: python>]>
 
+You can also use
+:meth:`~django.contrib.contenttypes.fields.GenericRelation.add`,
+:meth:`~django.contrib.contenttypes.fields.GenericRelation.create`, or
+:meth:`~django.contrib.contenttypes.fields.GenericRelation.set` to create
+relationships.
```

Comment:
`django.contrib.contenttypes.fields.GenericRelation` are not documented so it will not work. We can remove it and use simply ``` ``add()``, ``create()``, or ``set()`` ```
```suggestion
You can also use ``add()``, ``create()``, or ``set()`` to create
relationships::
```